packages/RichTextEditing-Core.package/RichTextDocument.class/instance/wordcount.st

```diff
@@ -1,16 +1,15 @@
-accessing
+count
 wordcount
 	
-	| byteString splitChars count i readWord|
+	| byteString count i readWord|
 	count := 0.
 	readWord := false.
 	byteString := self text string asByteArray.
-	splitChars := #(182 32 10 13).
 	
 	i := 1.
 	[i <= byteString size] whileTrue:[
-		(readWord and: (splitChars includes: (byteString at: i))) ifTrue: [readWord := false. count := count+1].
-		(splitChars includes: (byteString at: i)) ifFalse: [readWord := true].
+		(readWord and: (self emptyChars includes: (byteString at: i))) ifTrue: [readWord := false. count := count+1].
+		(self emptyChars includes: (byteString at: i)) ifFalse: [readWord := true].
 		i := i + 1
 		].
```

Comment:
just tested this small code snippet which checks if the previous element is an emptyChar and the current not:
```
| prev bool |
prev := 32.
^ (#[104 97 108 32 108 32 32 111] count: [:each | bool := ((#[32] includes: each) not and: (#[32] includes: prev)). prev := each. bool ]) 
```
Not sure if better, however more smalltalk like